packages/create-svelte/package.json

```
@@ -33,7 +33,7 @@
 		"lint": "eslint --ignore-path .gitignore --ignore-path ../../.gitignore \"./*.js\" && npm run check-format",
 		"format": "npm run check-format -- --write",
 		"check-format": "prettier --check . --config ../../.prettierrc --ignore-path ../../.gitignore --ignore-path .gitignore --plugin prettier-plugin-svelte --plugin-search-dir=.",
-		"prepublishOnly": "npm run build"
+		"prepublishOnly": "npm run build",
```

this is invalid JSON, no?

```suggestion
		"prepublishOnly": "npm run build"
```